src/core/src/op/strided_slice.cpp

```diff
@@ -172,7 +172,25 @@ AxisSet StridedSlice::convert_mask_to_axis_set(const std::vector<int64_t>& mask)
 
 std::shared_ptr<Node> StridedSlice::clone_with_new_inputs(const OutputVector& new_args) const {
     OV_OP_SCOPE(v1_StridedSlice_clone_with_new_inputs);
-    check_new_args_count(this, new_args);
+    // check_new_args_count(this, new_args);
+    NODE_VALIDATION_CHECK(this,
+                          (new_args.size() == 3) || (new_args.size() == 4),
+                          "clone_with_new_inputs() expected 3 or 4",
+                          " arguments",
+                          " but got ",
+                          new_args.size());
```

Comment:
```suggestion
    const auto args_size = new_args.size()
    NODE_VALIDATION_CHECK(this,
                          (args_size  == 3) || (args_size  == 4),
                          "Incorrect number of new inputs: ", args_size);
```
This `(args_size  == 3) || (args_size  == 4)` will be in error message number is known and `clone_with_new_inputs()` is not required the error message pointe place in code of this error.